apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -518,482 +524,510 @@ export const SignedVerdictOverview: React.FC = () => {
   }
 
   return (
-    <PageLayout
-      workingCase={workingCase}
-      activeSection={Sections.CASE_CLOSED}
-      isLoading={isLoadingWorkingCase}
-      notFound={caseNotFound}
-    >
-      <PageHeader
-        title={formatMessage(titles.shared.closedCaseOverview, {
-          courtCaseNumber: workingCase.courtCaseNumber,
-        })}
-      />
-      <FormContentContainer>
-        <Box marginBottom={5}>
-          <Box marginBottom={3}>
-            <Button
-              variant="text"
-              preTextIcon="arrowBack"
-              onClick={() => router.push(constants.CASES_ROUTE)}
-            >
-              Til baka
-            </Button>
-          </Box>
-          <Box display="flex" justifyContent="spaceBetween" marginBottom={3}>
-            <Box>
-              <Box marginBottom={1}>
-                <Text as="h1" variant="h1">
-                  {titleForCase(formatMessage, workingCase)}
-                </Text>
-              </Box>
+    <>
+      {workingCase.courtEndTime &&
+        !workingCase.isAppealDeadlineExpired &&
+        user?.role &&
+        isProsecutionRole(user.role) &&
+        features.includes(Feature.APPEAL_TO_COURT_OF_APPEALS) && (
```

Comment:
Consider hiding the old `AppealSection` below when the opposite is true, i.e. when `!features.lincludes(Feature.APPEAL_TO_COURT_OF_APPEALS)`.

Otherwise, court users will be able to appeal the old way even after the new feature is turned on.